src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Tab } from '../../types/Tab';
+import { TabInfo } from '../TabInfo';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const [selectedTab, setSelectedTab] = useState(
+    tabs.find((tab) => tab.id === selectedTabId) || tabs[0],
```

Comment:
you can put it in a separate variable

Reply:
agree
if you create a separate func - you can reuse it here